staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/celcoststability_test.go

```diff
@@ -241,6 +241,12 @@ func TestCelCostStability(t *testing.T) {
 				"!('x' in self.val1)":                            6,
 				"self.val1 + self.val2 == ['a', 'b', 'c']":       6,
 				"self.val1 + ['c', 'd'] == ['a', 'b', 'c', 'd']": 4,
+				"sets.contains(self.val1, ['a'])":                6,
+				"sets.equivalent(self.val1, ['a', 'b', 'c'])":    21,
+				"sets.intersects(self.val1, ['a'])":              6,
+				"sets.contains(self.val3, [1])":                  1003,
+				"!sets.equivalent(self.val3, [1, 2, 3])":         6004,
+				"sets.intersects(self.val3, [1])":                1003,
```

Comment:
Let's have at least 2 tests for each of `contains`, `equivalent` and `intersects`  that test different lengths of inputs (e.g. one that does `[1]` and one that does `[1, 2, 3....]`) and show that the cost changes in the way we expect for each function

Reply:
nvm, I missed that the val3 cases use the buildLargeArray input

Reply:
Thanks! I'll resolve this one